plugins/kubernetes/app/models/kubernetes/template_filler.rb

```diff
@@ -85,6 +85,20 @@ def self.dig_path(path)
 
     private
 
+    def namespaced_kind?(kind)
+      cluster = @doc.deploy_group.kubernetes_cluster
+      api_version = template.fetch(:apiVersion)
+
+      resources = Rails.cache.fetch(["template-filler-resources", api_version, cluster]) do
```

Comment:
how often cache is refreshed?
I.e. how to handle CRD updates?

Reply:
apparently once per day per cluster per api version. should be "good enough"